src/file/file.controller.ts

```diff
@@ -108,11 +108,11 @@ export class FileController {
       if (typeof raw === 'string' || Buffer.isBuffer(raw)) {
         await fs.promises.access(path.dirname(file), W_OK);
         await fs.promises.writeFile(file, raw);
-        return 'File uploaded successfully';
+        return `File uploaded successfully at ${file}`;
       }
     } catch (err) {
       this.logger.error(err.message);
-      throw err;
+      throw err.message;
```

Comment:
@Kif-Fando , is it on purpose? Better to throw error object rather than string

Reply:
@gmesserman I am stuck. From one side we don't want to throw any excess info about error. From other side we have app with huge amount of vulnerabilities.
I mean in that situation I want to keep _error.message_ to show only text of error without additional info where exactly it happened.